System.IO.FileSystem/File.cs

```diff
@@ -54,49 +41,35 @@ public static class File
 
             if (string.IsNullOrEmpty(destFileName))
             {
-                throw new ArgumentException(nameof(sourceFileName));
+                throw new ArgumentException(nameof(destFileName));

```

Comment:
The pattern used on these is to NOT pass the parameter names in order to save flash space.
I know it was there before, but can you please remove the nameof here and on the previous one?